src/app/models.py

```diff
@@ -86,10 +86,28 @@ class Sequence(SQLModel, table=True):
     pose_id: Union[int, None] = Field(None, foreign_key="poses.id", nullable=True)
     azimuth: float = Field(..., ge=0, lt=360)
     is_wildfire: Union[AnnotationType, None] = None
+    cone_azimuth: Union[float, None] = Field(None, nullable=True)
+    cone_angle: Union[float, None] = Field(None, nullable=True)
```

Comment:
Should not we go for `angular_width`? Also, this one must be in degree, hence a suggestion below


```suggestion
    angular_width: Union[float, None] = Field(None, ge=0, lt=360, nullable=True)
```

(modification must be done elsewhere to adapt the naming)